ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/proxy/workers-rhcos-rhel8/provision/cucushift-installer-rehearse-azure-ipi-proxy-workers-rhcos-rhel8-provision-chain.yaml

```diff
@@ -17,10 +17,10 @@ chain:
     default: "Standard_D4s_v3"
     documentation: The Size of the RHEL VM provisioned.
   - name: RHEL_IMAGE
-    default: "RedHat:RHEL:8-lvm-gen2:8.5.2021121504"
+    default: "RedHat:RHEL:8-lvm-gen2:8.6.2022051814"
```

Comment:
RedHat:RHEL:8-lvm-gen2:8.6.2022102701

Reply:
I guess this is fine as they're actually the same RHEL major version.